themes/tdr/login/messages/messages_en.properties

```diff
@@ -1,9 +1,14 @@
 # Standard Keycloak template message TDR overrides
-continueButton=Continue
+signInButton=Sign in
 doLogIn=Sign in
-loginTitle=Sign in
+loginTitle=Sign in to your account
+loginDescription=Please sign in to your account using the details provided in your invite email.
```

Comment:
What details are provided in the email? I just want to be sure it all makes sense in context.

(Also, is "invite email" right? Using "invite" as a noun sounds slightly too informal for an official service, but I might be old!)

Reply:
I have used Ashley's copy for this. I assumed this was signed off by Kirsten but I can check.

Reply:
Ashley said we can use the term 'invitation email' to add a more formal tone. Does that sound good to you?

Reply:
I haven't got a reply to the other question, is that maybe a keycloak thing and something that Tom knows?

Reply:
From talking to Kirsten, the invitation email is a separate email that we'll send to the users manually. I think it would be worth having a conversation with her about what will appear in the initial Keycloak email (which Tom knows), and what needs to go into the invitation email.